libnavigation-core/src/main/java/com/mapbox/navigation/core/test/TripServiceActivity.kt

```diff
@@ -0,0 +1,115 @@
+package com.mapbox.navigation.core.test
+
+import android.os.Bundle
+import android.os.SystemClock
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.annotation.module.MapboxModuleType
+import com.mapbox.base.common.logger.Logger
+import com.mapbox.base.common.logger.model.Message
+import com.mapbox.base.common.logger.model.Tag
+import com.mapbox.common.module.provider.MapboxModuleProvider
+import com.mapbox.navigation.base.TimeFormat.TWENTY_FOUR_HOURS
+import com.mapbox.navigation.base.internal.VoiceUnit.METRIC
+import com.mapbox.navigation.base.internal.extensions.inferDeviceLocale
+import com.mapbox.navigation.base.options.NavigationOptions
+import com.mapbox.navigation.base.trip.notification.TripNotification
+import com.mapbox.navigation.core.R
+import com.mapbox.navigation.core.Rounding
+import com.mapbox.navigation.core.internal.MapboxDistanceFormatter
+import com.mapbox.navigation.core.trip.service.MapboxTripService
+import com.mapbox.navigation.utils.internal.ThreadController
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
+import kotlinx.coroutines.launch
+
+internal class TripServiceActivity : AppCompatActivity() {
```

Comment:
```suggestion
internal class TripServiceTestActivity : AppCompatActivity() {
```

Reply:
I think it's ok, because it's inside `test` package. 
And it's used in `TripServiceActivityTest`.

If we rename it to `TripServiceTestActivity`, we should also rename the test to `TripServiceTestActivityTest`. It doesn't sound well ))